iml-manager-cli/src/main.rs

```diff
@@ -35,8 +36,12 @@ pub enum App {
         command: filesystem::FilesystemCommand,
     },
     #[structopt(name = "update_repo")]
-    ///  Update Agent repo files
+    /// Update Agent repo files
     UpdateRepoFile(update_repo_file::UpdateRepoFileHosts),
+
+    #[structopt(name = "debugapi", setting = structopt::clap::AppSettings::Hidden)]
```

Comment:
Maybe shorter `api` ?

Reply:
I want it clear this isn't meant as a user facing, it's for debug and testing.